lib/kitchen/directions/bake_chapter_review_exercises/v1.rb

```diff
@@ -4,12 +4,16 @@ module Kitchen::Directions::BakeChapterReviewExercises
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:, append_to:)
-      @metadata = metadata_source
-      @exercise_clipboard = Kitchen::Clipboard.new
+    def bake(chapter:, metadata_source:, append_to:, klass:)
+      @klass = klass
+      @metadata = metadata_source.children_to_keep.copy
+      @title = I18n.t(:"eoc.#{klass}")
+      @content = ''
```

Comment:
do you set this variable anywhere else? if not, line 34 is enough and you can remove this

Reply:
ohhhhh awesome thanks